File: Cpp/Height Checker.cpp
// 4ms, 63.30%; 7.7MB, 100.00%
class Solution
{
public:
    int heightChecker(const vector<int> &heights)
    {
        vector<int> tmp(heights);
        sort(tmp.begin(), tmp.end());
        int count = 0;
        for (int i = 0; i < heights.size(); ++i)
            if (heights[i] != tmp[i]) count += 1;
        return count;
    }
};

File: Python3/Height Checker.py
# 36ms, 45.68%; 12.8MB, 100.00%
class Solution:
    def heightChecker(self, heights: List[int]) -> int:
        heights2 = sorted(heights)
        return sum([heights[i] != heights2[i] for i in range(len(heights))])
        
